scripts/check_deps.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
sh?

Reply:
Do we need it to work on Windows?

Reply:
I don't think it's a requirement. There are just helpers anyway.